### PR TITLE
3.12.2

### PR DESCRIPTION
**Pull request by rasel-sk**
new: Slovak support

### CODE REVIEW

**Comment by rasel-sk** [comment]
test

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+3.12.2 / 2018-03-19
```
Comment (EvanHahn) [inline HISTORY.md]: Could you change this from "3.12.2 / 2018-03-19" to "UNRELEASED"?

File: humanize-duration.js [hunk 2]
```diff
@@ -540,6 +551,19 @@
     }
   }
 
+    // Internal helper function for Slovak language.
+  function getSlovakForm (c) {
```
Comment (EvanHahn) [inline humanize-duration.js]: This is identical to the `getCzechForm` helper. Could you delete this one, rename `getCzechForm` to something else (like `getCzechOrSlovakForm`), and use that one everywhere?

File: HISTORY.md (hunk 1 above)
Comment (rasel-sk) [reply HISTORY.md]: done!

File: humanize-duration.js (hunk 2 above)
Comment (rasel-sk) [reply humanize-duration.js]: done!

**Comment by EvanHahn** [comment]
This looks good! I will merge soon.

**Comment by EvanHahn** [comment]
_Ďakujem_! Published this in `humanize-duration@3.13.0`.

**Pull request merged**